follow-redirect-url.js

```diff
@@ -1,6 +1,9 @@
 "use strict";
 
 const fetch = require("node-fetch");
+const https = require("https");
+const http = require("http");
+fetch.Promise = Promise;
```

Comment:
`fetch.Promise = Promise;` is not required here fetch already has inbuilt promise.

Reply:
Removed! Not sure how this made its way in.